integration/integration_test.go

```diff
@@ -2349,6 +2351,22 @@ func testTrustedClusters(t *testing.T, suite *integrationTestSuite) {
 	trustedClusters(t, suite, trustedClusterTest{multiplex: false})
 }
 
+// testDisabledTrustedClusters tests creation of disabled trusted cluster
+func testDisabledTrustedClusters(t *testing.T, suite *integrationTestSuite) {
```

Comment:
This test fails for me.

```
    --- FAIL: TestIntegrations/TrustedDisabledClusters (5.11s)
        trustedclusters.go:214:
            	Error Trace:	/Users/zmb/teleport/integration/trustedclusters.go:214
            	            				/Users/zmb/teleport/integration/integration_test.go:2767
            	            				/Users/zmb/teleport/integration/integration_test.go:2359
            	            				/Users/zmb/teleport/integration/integration_test.go:97
            	Error:      	Received unexpected error:
            	            	access denied to zmb connecting to 127.0.0.1 on cluster cluster-aux
            	Test:       	TestIntegrations/TrustedDisabledClusters
```

Reply:
I have not been able to replicate this. Is this still the case with the latest version of this branch?

Reply:
This test fails for me periodically as well albeit with a different error:

```
--- FAIL: TestIntegrations (8.16s)
    --- FAIL: TestIntegrations/TrustedDisabledClusters (8.15s)
        trustedclusters.go:206:
            	Error Trace:	/Users/r0mant/code/go/src/github.com/gravitational/teleport/integration/trustedclusters.go:206
            	            				/Users/r0mant/code/go/src/github.com/gravitational/teleport/integration/integration_test.go:2771
            	            				/Users/r0mant/code/go/src/github.com/gravitational/teleport/integration/integration_test.go:2366
            	            				/Users/r0mant/code/go/src/github.com/gravitational/teleport/integration/integration_test.go:99
            	Error:      	Condition never satisfied
            	Test:       	TestIntegrations/TrustedDisabledClusters
            	Messages:   	Two clusters cannot connect to each other
```

We need to fix the flakiness before we can merge this.

Reply:
I think the previous max timeout for the eventually was just far too small. I have upped the amount of time it will wait before giving up

Reply:
I just pulled the latest and am still seeing the error Roman reported.

```
--- FAIL: TestIntegrations (12.38s)
    --- FAIL: TestIntegrations/TrustedDisabledClusters (12.38s)
        trustedclusters.go:206:
            	Error Trace:	/Users/zmb/teleport/integration/trustedclusters.go:206
            	            				/Users/zmb/teleport/integration/integration_test.go:2771
            	            				/Users/zmb/teleport/integration/integration_test.go:2366
            	            				/Users/zmb/teleport/integration/integration_test.go:99
            	Error:      	Condition never satisfied
            	Test:       	TestIntegrations/TrustedDisabledClusters
            	Messages:   	Two clusters cannot connect to each other
```

Additionally, it looks like the runtime of the test more than doubled, from 5 seconds to  12.

Reply:
@EdwardDowling I've done more troubleshooting here and I'm pretty confident this behavior is due to caching as debug logging shows that cache removes cluster-main's user authority for some reason. Disabling caching makes the test pass with `-count 10` for me. I'm not sure there's a reliable way to rewrite this test that disables/enables authorities with our caching system. Let's disable the cache just for this particular test.

Reply:
I've added the lines for disabling the caching for this test.